nautobot/core/settings.py

```diff
@@ -659,7 +657,7 @@
 # Branding URLs (links in the bottom right of the footer)
 BRANDING_URLS = {
     "code": os.getenv("NAUTOBOT_BRANDING_URLS_CODE", "https://github.com/nautobot/nautobot"),
-    "docs": os.getenv("NAUTOBOT_BRANDING_URLS_DOCS", "https://nautobot.readthedocs.io/"),
+    "docs": os.getenv("NAUTOBOT_BRANDING_URLS_DOCS", f"{STATIC_URL}docs/index.html"),
```

Comment:
Can this use the `static` templatetag function so as to ensure correct behavior when using staticfiles_storage? Or is that not available at this point in initialization?

Reply:
I can check on this.